blueflood-core/src/main/java/com/rackspacecloud/blueflood/types/BluefloodEnumRollup.java

```diff
@@ -41,6 +41,16 @@ public int getCount() {
         return hashedEnumValues2Count.size();
     }
 
+    public int getNumPoints() {
+        int enumCount = 0;
+        Set keySet = hashedEnumValues2Count.keySet();
+        Iterator itr = keySet.iterator();
+        while(itr.hasNext()){
+            enumCount+= hashedEnumValues2Count.get(itr.next());
+        }
```

Comment:
wouldn't it be clearer to iterate through the values with the foreach syntax, something like this:
for (Long value: hashedEnumValues2Count.values()):
    enumCount += value;